scan.go

```diff
@@ -238,6 +239,16 @@ func scan(pkgPatterns []string) ([]*unit.SourceUnit, error) {
 		// compare results in tests more easily.
 		pkg.Root = ""
 
+		if strings.HasPrefix(pkg.ImportPath, "_") {
+			wd, err := os.Getwd()
+			if err != nil {
+				return nil, err
+			}
+			pkg.ImportPath = repo + pkg.ImportPath[1+len(wd):]
```

Comment:
Is a solution to modify `go list ...` to follow symlinks? What do you think of the overall reasoning of the comment (better to modify `GOPATH` at the top level and run everything as usual instead of adding local changes to check for being outside of `GOPATH`)?